### PR TITLE
Swipe Refresh

### PR DESCRIPTION
**Pull request by javiergamarra**
Feel free to change whatever you need :)

### CODE REVIEW

**Comment by mdelapenya** [comment]
Just started reviewing :) <br><br>:octocat: _Sent from [GH](http://nodegh.io)._

File: app/src/main/java/es/craftsmanship/toledo/katangapp/activities/ShowStopsActivity.java [hunk 1]
```diff
@@ -27,18 +26,15 @@
...
     private RecyclerView recyclerView;
     private SwipeRefreshLayout swipeRefreshLayout;
 
     @Subscribe
     public void busStopsReceived(QueryResult queryResult) {
         List<BusStopResult> results = queryResult.getResults();
 
-        busStopResults.addAll(results);
-
-        recyclerView.setLayoutManager(new LinearLayoutManager(this));
```
Comment (mdelapenya) [inline app/src/main/java/es/craftsmanship/toledo/katangapp/activities/ShowStopsActivity.java:40]: Isn't this line required? It's used in L69

Comment (javiergamarra) [inline app/src/main/java/es/craftsmanship/toledo/katangapp/activities/ShowStopsActivity.java:40]: No, we aren't resetting the state of the recyclerView, just of the adapter. The recycler view is configured at the start to be a vertical list (linearlayoutmanager). We only need to change it or set it again if we want to show an horizontal list or a grid.

Comment (mdelapenya) [inline app/src/main/java/es/craftsmanship/toledo/katangapp/activities/ShowStopsActivity.java:40]: Got it!

**Comment by mdelapenya** [comment]
Pull request submitted to https://github.com/craftsmanship-toledo/katangapp-android/pull/50. [See changes here.](https://github.com/mdelapenya/katangapp-android/compare/693432c~1...693432c) <br><br>:octocat: _Sent from [GH](http://nodegh.io)._

**Comment by mdelapenya** [comment]
Pull request submitted to https://github.com/craftsmanship-toledo/katangapp-android/pull/51. [See changes here.](https://github.com/mdelapenya/katangapp-android/compare/693432c~1...693432c) <br><br>:octocat: _Sent from [GH](http://nodegh.io)._

**Comment by mdelapenya** [comment]
Thank you very much @nhpatt for this review. It fixed instability on the list redrawn ;)

**Pull request closed without merging**